### PR TITLE
add capistrano deploy

### PR DESCRIPTION
**Pull request by ndushay**
## Why was this change made?

allow capistrano deploys to deploy VM.

## How was this change tested?

did a cap deploy and it works

## Which documentation and/or configurations were updated?

xxx

### CODE REVIEW

**Comment by jcoyne** [comment]
Isn’t Capistrano a pretty heavyweight way to do this? It’s not running any services. Why not just `git pull`?

**Comment by ndushay** [comment]
1.  I made the temp directory a linked directory (or so I thought).
2.  why not use the same mechanism we always use for deployment?  I kept cap stuff small.

**Comment by jcoyne** [comment]
@ndushay this project feels to me like a totally different thing than our other projects.  Typically we deploy services, migrate databases, update shared configs, restart servers and we keep a number of releases and we want the ability to rollback to a previous release.  That is what capistrano does well, we don't need any of that.  This is just a single ruby script and all you never need to deploy it more than one time.  It didn't even have a gemfile that we need to keep updated until we added capistrano.  By adding this dependency, we make the process slower, more complex and add maintenance overhead.

File: .gh_access_token [hunk 1]
```diff
@@ -0,0 +1 @@
+30759564cd7806418bd8bd5d538fceb5299d27f5
```
Comment (jcoyne) [inline .gh_access_token:1]: You probably want to burn this token.

File: deploy.rb [hunk 2]
```diff
@@ -2,17 +2,22 @@
 # frozen_string_literal: true
 
 $LOAD_PATH.unshift File.expand_path('lib', __dir__)
+
+require 'rubygems'
+require 'bundler/setup'
+Bundler.require(:default)
```
Comment (jcoyne) [inline deploy.rb:8]: What is this doing for us?

**Comment by ndushay** [comment]
At this point, I am no longer trying to run this code off the sdr-deploy VM.  

One concern:
- how do we get the aliased "deploy" user on sdr-deploy to have the appropriate ssh access to our git repos (or would we have to change the script to use https instead)?

**Comment by mjgiarlo** [comment]
@ndushay worth keeping this PR around? or can we nuke it?

**Pull request closed without merging**